components/upload/text/TextBox.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useRef, useState} from 'react';
+import { View, TextInput, Text} from 'react-native';
+import { BlurView } from 'expo-blur';
+
+type TextBoxProps = {
+    text: string,
+    setText: (text: string) => void,
+    prompt: string,
+    viewPromptOpen: boolean,
+}
+
+//Keep in Mind: May need to fix text box limit logic for people who set their text to be larger font size
```

Comment:
Thanks for adding these comments! Will be very helpful for all the changes that we might need to take care of for the future.